src/components/Footer/style.module.scss

```diff
@@ -0,0 +1,71 @@
+@use '@/styles/vars' as vars;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  justify-content: center;
+  align-items: center;
+  padding: 8rem vars.$side-padding;
+  position: relative;
+  margin: 0 calc(vars.$side-padding * -1);
```

Comment:
> footer still stretches body out because of the `margin: 0 -4rem`

I think it's because `vars.$side-padding` isn't actually constant; it's less than 4em on mobile. Maybe it should be a CSS variable instead, set by breakpoints, and then use the CSS variable here and for the body's horizontal padding